src/DI.php

```diff
@@ -71,13 +71,20 @@ public static function getDice()
 	//
 
 	/**
+	 * @deprecated 2024.12 use DI::apphelper() instead
+	 *
 	 * @return App
 	 */
 	public static function app()
 	{
 		return self::$dice->create(App::class);
 	}
 
+	public static function apphelper(): AppHelper
```

Comment:
Please rename to snakeCase:
```suggestion
	public static function appHelper(): AppHelper
```

Reply:
Done.